fontforge/sfd.c

```diff
@@ -87,6 +88,7 @@ static const char *caps[] = { "butt", "round", "square", "inher", NULL };
 static const char *spreads[] = { "pad", "reflect", "repeat", NULL };
 
 int prefRevisionsToRetain = 32;
+int WritePNGInSFD = true;
```

Comment:
Man, I really dislike our prefs system, it's so bad.

Rant aside, can we not Pascal case this, we generally reserve that for functions

Reply:
The reason I want to preserve it is, that's the name of the config option used in the interface and the file...in `prefs.c` it seems pretty much random if that's respected. `extern int OpenCharsInNewWindow` maps to option `OpenCharsInNewWindow`&hellip;that makes sense to me. `extern int ask_user_for_resolution` maps to `AskBDFResolution`&hellip;that makes none, it just adds confusion using a different name.

Reply:
That the UI displays something different is what I would expect to be honest. 

I still have a preference for this to be `write_png_in_sfd`; as is, this looks like a function.

Reply:
@jtanx, it sounds like you don't feel terribly strongly about this? Can we mark it resolved?

Reply:
I would still prefer it, because (I think) pascal case in this code base should be reserved for functions, but the reality of it is we're already not too consistent with how we name those preferences.